apps/src/templates/VersionHistory.jsx

```diff
@@ -132,7 +132,7 @@ var VersionHistory = React.createClass({
                   </td>
                   <td width="250" style={{textAlign: 'right'}}>
                   <button className="btn-danger" onClick={this.onConfirmClearPuzzle} style={{float: 'right'}}>
-                    Delete Progress
+                    Start over
```

Comment:
Should this be internationalizable?

Reply:
applab/gamelab are currently not internationalized

Reply:
Got it!